.gitignore

```diff
@@ -1,2 +1,2 @@
-book
-.blacksmith-cache.json
```

Comment:
Why would `.blacksmith-cache.json` be removed? Don't we still generate this file?

Reply:
Guess it ges loss on rebase

Reply:
Done.